src/lib/asn1/asn1_obj.h

```diff
@@ -444,7 +444,10 @@ class BOTAN_PUBLIC_API(2,0) AlgorithmIdentifier final : public ASN1_Object
       const OID& oid() const { return m_oid; }
```

Comment:
For consistency with #3264: shouldn't this be `object_identifier()`?

Reply:
That would be better. I'll rework this.